From 41d80cbce5f07bd9a2406cc54af23a200e4409e4 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Thu, 30 Jul 2015 16:33:53 -0500
Subject: [PATCH 0743/5242] MLK-11334: ARM: dts: enable qspi IOMUX sleep state
 for imx7d-12x12-lpddr3-arm2

commit  d3c5c17de89e41c266cb9687389f9e957bc255c8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable qspi IOMUX sleep state for imx7d-12x12-lpddr3-arm2.

Signed-off-by: Han Xu <b45815@freescale.com>
(cherry picked from commit 73773c936b0c33e6e01e14c646749dfa3daaedad)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
index ede6df2..3a7af2e 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
@@ -39,8 +39,9 @@
 };
 
 &qspi1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_qspi1_1>;
+	pinctrl-1 = <&pinctrl_qspi1_1>;
 	status = "okay";
 	fsl,qspi-has-second-chip = <1>;
 	ddrsmp=<0>;
-- 
1.7.9.5

